A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -46,6 +46,18 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
         }
         this._onModeManagedHeading();
         this._onModeManagedAltitude();
+
+        let thrRedAccAltitude;
```

Comment:
It's a matter of style, but this can be written with less lines and without mutable variable. And then encapsulated into a function in `A320_Neo_CDU_InitPage.js` to avoid copy-paste.

Like
```
static updateThrRedAcc(mcdu) {
    const origin = mcdu.flightPlanManager.getOrigin();
    const thrRedAccAltitude = origin && origin.altitudeinFP ? origin.altitudeinFP + 1500 : undefined;

    mcdu.thrustReductionAltitude = thrRedAccAltitude;
    mcdu.accelerationAltitude = thrRedAccAltitude;
}
static ShowPage1(mcdu) { // existing code
[...]
```
and then call `CDUInitPage.updateThrRedAcc(this);` from this file and `CDUInitPage.updateThrRedAcc(mcdu);` from file above.